scripts/globals/weaponskills.lua

```diff
@@ -250,7 +251,29 @@ local function getMultiAttacks(attacker, target, wsParams)
         end
     end
 
-    numHits = utils.clamp(numHits + bonusHits, 1, 8)
+    -- If this WS/Ability is capable of multi-striking from Occasionally Attacks X weapons, calculate now
```

Comment:
I think in most cases you cannot have both bonusHits and totalOAXHIts apply (see https://www.ffxiah.com/forum/topic/31534/math-oax-multi-attack-and-you/#1926857 and also the normal attack code for this) so if bonusHits is >0 then totalOAXHits should be 0.

Reply:
Ah I think you are correct; that was my misunderstanding of ["Occasionally attacks X times" statistics are processed after [Quadruple Attack](https://www.bg-wiki.com/ffxi/Quadruple_Attack), [Triple Attack](https://www.bg-wiki.com/ffxi/Triple_Attack), and [Double Attack](https://www.bg-wiki.com/ffxi/Double_Attack) checks](https://www.bg-wiki.com/ffxi/Occasionally_Attacks_X_Times). I'll fix it.

Reply:
Can just be bonusHits == 0 I think?

Reply:
Updated